src/sentry/integrations/issues.py

```diff
@@ -96,6 +96,57 @@ def get_link_issue_config(self, group, **kwargs):
             }
         ]
 
+    def get_persisted_default_config_fields(self):
+        """
+        Returns a list of field names that should have their last used values
+        persisted on a per-project basis.
+        """
+        return []
+
+    def store_issue_last_defaults(self, project_id, data):
+        """
+        Stores the last used field defaults on a per-project basis. This
+        accepts a dict of values that will be filtered to keys returned by
+        ``get_persisted_default_config_fields`` which will automatically be
+        merged into the associated field config object as the default.
+         >>> integ.store_issue_last_defaults(1, {'externalProject': 2})
+         When the integration is serialized these values will automatically be
+        merged into the field configuration objects.
+         NOTE: These are currently stored for both link and create issue, no
+              differentiation is made between the two field configs.
+        """
```

Comment:
What happened to the comment :(

Reply:
https://github.com/getsentry/sentry/blob/cd099dd6e0a7d7017fe2ba88211a03ddfc5b0226/src/sentry/integrations/issues.py#L107-L120

Reply:
Honestly I'm not sure.... :/ maybe my linter decided to cause it harm. I'll put it back to what it was.